python/apps/taiga/tests/integration/taiga/projects/projects/test_api.py

```diff
@@ -39,7 +40,7 @@ async def test_create_project_being_no_workspace_member(client):
     assert response.status_code == status.HTTP_403_FORBIDDEN, response.text
 
 
-async def test_create_project_being_anonymous(client):
+async def test_create_project_200_ok_being_anonymous(client):
```

Comment:
This test returns  status.HTTP_403_FORBIDDEN

Reply:
Yes, it's wrong. I'll fix it.